### PR TITLE
Create LIT7021BaryaAganentGudale.xml

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
A prayer attested in Abb. 207, poorly rescribed.

### CODE REVIEW

File: new/LIT7021BaryaAganentGudale.xml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7021BaryaAganentGudale" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Protective prayer against bāryā, demons, desk, gudāle and budā</title>
```
Comment (CarstenHoffmannMarburg) [inline new/LIT7021BaryaAganentGudale.xml:8]: Dasak? I wonder, if bāryā and dasak are better to be seen as demonic beings, as well. I think, they should be given in capitals in the title? Cf. LIT6856PPHemamaBarya (Prayer against Bāryā and Legewon). E.g. "Bāryā, demons, Dasak (or: Dask), gudāle (or: gudāle) and budā".

**Review by CarstenHoffmannMarburg — CHANGES_REQUESTED** [review]

File: new/LIT7021BaryaAganentGudale.xml (hunk 1 above)
Comment (eu-genia) [inline new/LIT7021BaryaAganentGudale.xml:8]: lang=`"en"` for translated titles, gez = fidel

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/LIT7021BaryaAganentGudale.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply new/LIT7021BaryaAganentGudale.xml:8]: OK

Comment (DenisNosnitsin1970) [reply new/LIT7021BaryaAganentGudale.xml:8]: Sorry! corrected.

**Review by CarstenHoffmannMarburg — APPROVED** [review]

**Pull request merged**